plugins/video-toggle/back.js

```diff
@@ -0,0 +1,11 @@
+const { injectCSS } = require("../utils");
+const path = require("path");
+const { urlToHttpOptions } = require("url");
+
+module.exports = (win, options) => {
+    if (options.forceHide) {
+		injectCSS(win.webContents, path.join(__dirname, "forceHide.css"));
```

Comment:
nit: lower case file names are often better because some filesystems are case sensitive (eg Linux) and others are not (eg MacOS).
```suggestion
		injectCSS(win.webContents, path.join(__dirname, "force_hide.css"));
```